### PR TITLE
Add Pt-br

### PR DESCRIPTION
**Pull request by contateste584394857**
Add Portuguese - Brazil language to the bot

### CODE REVIEW

**Comment by contateste584394857** [comment]
@umutxyp

File: commands/language.js [hunk 1]
```diff
@@ -29,6 +29,10 @@ module.exports = {
           .setLabel("Français ")
           .setCustomId('fr')
           .setStyle(ButtonStyle.Secondary),
+          new ButtonBuilder()
+          .setLabel("Português ")
```
Comment (Fyxren) [inline commands/language.js:33]: There is a wild space here :)

**Review by Fyxren — APPROVED** [review]
Apart from a wild space in commands/language.js line 33, it looks fine

**Review by Fyxren — APPROVED** [review]
Solid stuff 👍

**Review by umutxyp — APPROVED** [review]

**Review by umutxyp — APPROVED** [review]

**Review by umutxyp — APPROVED** [review]

**Review by umutxyp — APPROVED** [review]

**Review by umutxyp — APPROVED** [review]

**Review by umutxyp — APPROVED** [review]

**Review by umutxyp — APPROVED** [review]

**Comment by contateste584394857** [comment]
Why is merging blocked

**Review by Fyxren — APPROVED** [review]
Looks good to me 👍

**Comment by Fyxren** [comment]
> Why is merging blocked

I am not sure to be honest, it mentions something about all commits having to be [signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification)

**Comment by Fyxren** [comment]
image

Seems like the first 2 commit weren't signed

**Comment by contateste584394857** [comment]
> image
> 
> Seems like the first 2 commits weren't signed

I commited the last one using github.com, and the first 2 with Git

**Comment by Fyxren** [comment]
> > Seems like the first 2 commits weren't signed
> 
> I commited the last one using github.com, and the first 2 with Git

Explains, I haven't had this issue before, so I don't really know a lot about it. Came across [this on StackOverflow](https://stackoverflow.com/questions/59351257/unverified-commits-in-github) tho. It may help, but maybe @umutxyp knows a work-around. In the end, he (_... or she, not sure_) is the owner of the repo

**Comment by contateste584394857** [comment]
#56

**Pull request closed without merging**